scripts/release-update-tilt-docs-repo.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Updates the Tilt docs repo with the latest version info.
+#
+# Usage:
+# scripts/update-docs-tilt-repo.sh $VERSION
+# where VERSION is of the form 0.1.0
+
+set -e
+
+if [[ "$GITHUB_TOKEN" == "" ]]; then
+    echo "Missing GITHUB_TOKEN"
+    exit 1
+fi
+
+VERSION=${1//v/}
+VERSION_PATTERN="[0-9]+\.[0-9]+\.[0-9]+"
+if ! [[ $VERSION =~ $VERSION_PATTERN ]]; then
+    echo "Version did not match expected pattern. Actual: $VERSION"
+    exit 1
+fi
+
+DIR=$(dirname "$0")
+cd "$DIR/.."
+
+ROOT=$(mktemp -d)
+git clone https://tilt-releaser:"$GITHUB_TOKEN"@github.com/tilt-dev/tilt.build "$ROOT"
```

Comment:
Hm, I think most of us have GITHUB_TOKEN set to tokens for our personal github accounts. This might be a naive question, but does this somehow work with those, or do we need to set it to a token for the tilt-releaser account?

Reply:
i think that's fine. eventually we'll run this on ci with some sort of a bot account and token. We've already been using personal GITHUB_TOKENs to release, e.g. https://github.com/tilt-dev/homebrew-tap